### DISCUSSION TITLE
Connecting to google drive folders - Radarr/Sonarr root folder parameter

### DISCUSSION THREAD

**Discussion by LeendertPlas** [meisnate12/Plex-Meta-Manager#846] [Q&A]
Is it possible to list radarr and sonarr drives that are connected with rclone? When i try, the drives seem to be listed weirdly and no variation of it seems to be working.

      arrapi.exceptions.Invalid: Invalid Root Folder: 'Z:\Plex\Films - 4K\' Options: [[6:U:\Plex\Films - 4K]] |

Then replacing the root folder with the option results in:

      arrapi.exceptions.Invalid: Invalid Root Folder: '[['6:U:\\Plex\\Films - 4K']]' Options: [[6:U:\Plex\Films - 4K]] |

No variation of changing the brackets or slashes results in a working version.

**Comment by meisnate12** [comment]
try using `U:\Plex\Films - 4K` and if that still doesnt work try `6`
